documentation/asciidoc/stories/assembly_creating_clusters.adoc

```diff
@@ -1,25 +1,15 @@
 ifdef::context[:parent-context: {context}]
 [id='creating-clusters']
-:context: start
-= Getting started with {ispn_operator}
-{ispn_operator} lets you create, configure, and manage {brandname} clusters.
-
-.Prerequisites
-
-* Install {ispn_operator}.
-//Community
-ifdef::community[]
-* Have an `oc` or a `kubectl` client.
-endif::community[]
-//Downstream
-ifdef::downstream[]
-* Have an `oc` client.
-endif::downstream[]
+:context: infinispan-cr
+= Getting started with Infinispan CR
+`Infinispan` Custom Resource (CR) instructs {ispn_operator} to create and manage {brandname} clusters on {k8s}.
```

Comment:
I'm not sure "instructs" is the correct term here. It's more of a specification that is reconciled by the operator in order to create a cluster matching those requirements. Not sure how to word that though :smile: